common/src/main/scala/nexusprovider/NexusConvertor.scala

```diff
@@ -1,22 +1,21 @@
 package nexusprovider
+
 import utils.{Convertor, UserModCommand}
 
 
 object NexusConvertor extends Convertor {
 
-  def convert (text: String): List[UserModCommand] = {
+  def convert(text: String): List[UserModCommand] = {
 
     val arr = text.split(",")
     val segmentsMap = arr
-      .view(3,arr.size)
-      .map { v =>
-      (v, arr(0))
+      .view(3, arr.size)
+      .map {
+        _ -> arr(0)
```

Comment:
Please avoid extended brackets if using undecore and simple expressions
Refactor to single line costruction